### PR TITLE
added http client to use the new fleet api

### PR DESCRIPTION
**Pull request by teemow**

### CODE REVIEW

File: client_cli.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+	if err != nil {
+		return errgo.Mask(err)
+	}
+
+	return nil
+}
+
+func (this *ClientCLI) Get(name string) (*job.Job, error) {
+	return &job.Job{}, nil
```
Comment (zeisss) [inline client_cli.go]: Please return an error here.

File: client_api.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	if err != nil {
+		return err
+	}
+	this.client.DestroyJob(j.Name)
+	
+	return nil
+}
+
+func (this *ClientAPI) Status(name string) (Status, error) {
+	return Status{}, nil
```
Comment (zeisss) [inline client_api.go]: please return an error

**Comment by zeisss** [comment]
besides the missing errors for the stubs this LGTM

File: examples/cli.go [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import (
+	"fmt"
+	"github.com/catalyst-zero/fleet-client-go"
+)
+
+func main() {
+	cAPI := client.NewClientAPI()
+
+	j, err := cAPI.Get("app.service")
```
Comment (xh3b4sd) [inline examples/cli.go:11]: Where does that `app.service` exist?

Comment (teemow) [inline examples/cli.go:11]: It's an example :)

**Pull request merged**